libraries.gradle

```diff
@@ -68,12 +68,10 @@ ext {
             slf4j_api:                 "org.slf4j:slf4j-api:1.7.5",
             slf4j_log4j:               "org.slf4j:slf4j-log4j12:1.7.5",
 
-            // jaxb task
-			jaxb:           'com.sun.xml.bind:jaxb-xjc:2.2.5',
-			jaxb2_basics:   'org.jvnet.jaxb2_commons:jaxb2-basics:0.6.3',
-			jaxb2_ant:      'org.jvnet.jaxb2_commons:jaxb2-basics-ant:0.6.3',
-			jaxb2_jaxb:     'org.jvnet.jaxb2_commons:jaxb2-basics-jaxb:2.2.4-1',
-			jaxb2_jaxb_xjc: 'org.jvnet.jaxb2_commons:jaxb2-basics-jaxb-xjc:2.2.4-1',
+            // JAXB task (using the task from JAXB Commons so we may benefit from the additional options it provides)
+            jaxb_xjc:               'org.glassfish.jaxb:jaxb-xjc:2.2.11',
+            jaxb2_commons_basics:   'org.jvnet.jaxb2_commons:jaxb2-basics:0.9.3',
+            jaxb2_commons_ant:      'org.jvnet.jaxb2_commons:jaxb2-basics-ant:0.9.3',
```

Comment:
Hm, apparently this was a transient thing.  I rebooted my computer (ugh) and this went away